### PR TITLE
Expose AKS custom headers variable

### PR DESCRIPTION
**Pull request by sumanthreddy29**
- [Custom headers defined in the pipeline](https://dev.azure.com/akstelescope/telescope/_build/results?buildId=941&view=logs&j=1bff13dd-d558-57b7-49f9-d18e9e1856ee&t=b6443cd2-8fab-5898-5144-71e4936682c6)
- [Without custom headers defined in the pipeline](https://dev.azure.com/akstelescope/telescope/_build/results?buildId=947&view=logs&j=1bff13dd-d558-57b7-49f9-d18e9e1856ee&t=47634be7-5853-5e81-d82e-084b76dee1c7)

### CODE REVIEW

File: steps/terraform/set-input-variables-azure.yml [hunk 1]
```diff
@@ -19,18 +19,27 @@ steps:
       SYSTEM_NODE_POOL=${SYSTEM_NODE_POOL:-null}
       USER_NODE_POOL=${USER_NODE_POOL:-null}
 
+      if [ -z "$(AKS_CUSTOM_HEADERS)" ]; then
+        CUSTOM_HEADERS='[]'
```
Comment (anson627) [inline steps/terraform/set-input-variables-azure.yml:23]: AKS_CUSTOM_HEADERS

File: steps/terraform/set-input-variables-azure.yml [hunk 2]
```diff
@@ -19,18 +19,27 @@ steps:
       SYSTEM_NODE_POOL=${SYSTEM_NODE_POOL:-null}
       USER_NODE_POOL=${USER_NODE_POOL:-null}
 
+      if [ -z "$(AKS_CUSTOM_HEADERS)" ]; then
```
Comment (anson627) [inline steps/terraform/set-input-variables-azure.yml:22]: AKS_CLI_CUSTOM_HEADERS

**Review by anson627 — DISMISSED** [review]

File: steps/terraform/set-input-variables-azure.yml (hunk 1 above)
Comment (sumanthreddy29) [reply steps/terraform/set-input-variables-azure.yml]: Updated

File: steps/terraform/set-input-variables-azure.yml (hunk 2 above)
Comment (sumanthreddy29) [reply steps/terraform/set-input-variables-azure.yml]: updated

**Review by anson627 — APPROVED** [review]

**Pull request merged**